mitwindfarm/Rotor.py

```diff
@@ -111,9 +111,9 @@ def __call__(self, x: float, y: float, z: float, windfield: Windfield, Ctprime,
         xs_glob, ys_glob, zs_glob = xs_loc + x, ys_loc + y, zs_loc + z
 
         # sample windfield and calculate rotor effective wind speed
-        Us, TIs = windfield.wsp_and_TI(xs_glob, ys_glob, zs_glob)
+        Us = windfield.wsp(xs_glob, ys_glob, zs_glob)
         REWS = self.rotor_grid.average(Us)
-        RETI = np.sqrt(self.rotor_grid.average(TIs**2))
+        RETI = np.mean(windfield.RETI(np.array([x]), np.array([y]), np.array([z])))
```

Comment:
unfortunately np.array([x]) won't work if x is a dual number. can this be removed?

Reply:
The reason I added the conversion to arrays in the call to RETI was that I wanted to make RETI a function that takes in ArrayLike coordinates. The ArrayLike format is necessary for the required slicing in centerline_wake_added_turb and RE_wake_added_turbulence. Perhaps we can replace this with a statement which checks if the coordinates are floats and if they are converts them to arrays.